raiden-ts/src/channels/types.ts

```diff
@@ -19,7 +23,7 @@ export type Lock = t.TypeOf<typeof Lock>;
  * because BP signature requires the hash of the message, for authentication of data not included
  * nor relevant for the smartcontract/BP itself, but so for the peers (e.g. payment_id)
  */
-export const SignedBalanceProof = t.type({
+export const BalanceProof = t.type({
```

Comment:
I am asking here, I might find it later but how are we handling the signed balance proofs now?